playwright-tests/util/rpcmock.js

```diff
@@ -542,29 +542,32 @@ export async function mockWithFTBalance({
       });
     }
   );
-  if (!isSufficient) {
-    await page.route(`https://rpc.mainnet.near.org`, async (route) => {
-      const request = await route.request();
-      const requestPostData = request.postDataJSON();
+  await page.route(`https://rpc.mainnet.near.org`, async (route) => {
+    const request = await route.request();
+    const requestPostData = request.postDataJSON();
 
-      if (
-        requestPostData.params &&
-        requestPostData.params.method_name === "ft_balance_of"
-      ) {
-        const json = {
-          jsonrpc: "2.0",
-          result: {
-            block_hash: "Hu4y62y9L8zvoSkeKHan3up7UWAxxuB9P2J9S89eMvp1",
-            block_height: 141681041,
-            logs: [],
-            result: [34, 49],
-          },
-          id: "dontcare",
-        };
-        await route.fulfill({ json });
-      } else {
-        await route.continue();
-      }
-    });
-  }
+    if (
+      requestPostData.params &&
+      requestPostData.params.method_name === "ft_balance_of"
+    ) {
+      const json = {
+        jsonrpc: "2.0",
+        result: {
+          block_hash: "Hu4y62y9L8zvoSkeKHan3up7UWAxxuB9P2J9S89eMvp1",
+          block_height: 141681041,
+          logs: [],
+          result: isSufficient
+            ? [
+                49, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
```

Comment:
Rather than just having bytes here, can you show JSON that you stringify and encode to bytes? it would be more readable then